spec/system/casework/viewing_an_application/that_is_open_and_unassigned_spec.rb

```diff
@@ -363,11 +363,6 @@
                             //tr[contains(td[1], 'test.pdf')]")
       expect(evidence_row).to have_content('test.pdf Download file (pdf, 12 Bytes)')
     end
-
-    it 'raises an error if user attempts to download the file' do
-      click_on 'Download file (pdf, 12 Bytes)'
-      expect(page).to have_content('You must assign this application to your list to download files')
-    end
```

Comment:
Not sure if it was worth duplicating the spec to check that a file successfully downloads if it's in the [assigned spec file](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/blob/main/spec/system/casework/viewing_an_application/that_is_assigned_to_me_spec.rb#L51)

Reply:
Nope not worth duplicating if it is testing the same thing.